README.md

```diff
@@ -137,3 +137,6 @@ source ~/.bashrc
 ## Discussion
 
 * For any questions reach out to us on any of the communication channels given on our website https://move2kube.konveyor.io/
+
+## contributer 
```

Comment:
@kushal9897 please phrase this section with a few lines and then point to the contributions document.